### DISCUSSION TITLE
Ability to collapse parent accounts in "Balance" view

### DISCUSSION THREAD

**Discussion by plibither8** [ananthakumaran/paisa#139] [Ideas]
Thanks a lot for your work Anantha! I'm slowly getting used to working around the app, and importing transactions.

I have over 200 stocks that I have purchased in the past, and each is a separate child account, as per the convention. It would be great to have the ability to collapse parent accounts, like `Assets:Stocks`. Here's a screenshot of my view for context: 
![Google Chrome 2024-01-11 at 23 35 52 - 001160](https://github.com/ananthakumaran/paisa/assets/31861755/a44002b5-4d68-4bae-9bdb-645561706dcc)

By collapsing, I mean something like the [`HTML <summary> tag`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary).

Once again, thanks so much! :)

**Comment by ananthakumaran** [comment]
It has been requested earlier as well, I will get to it when I get some time. Btw, you can use the `hide numbers` option in the right-hand side drop-down menu to hide all numbers if you want to share screenshots without sharing any numbers.

**Reply by plibither8** [reply]
Oh perfect, thanks a lot! Also, cool nifty tip with the hide all numbers! :)
